.devcontainer/devcontainer.json

```diff
@@ -3,7 +3,7 @@
 {
 	"name": "Python 3",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/python:0-3.8",
+	"image": "mcr.microsoft.com/devcontainers/python:0-3.10",
```

Comment:
Q: wondering if we should develop in the lowest supported version of python.  Running things like this we could introduce breaking changes that don't get picked up until the CI pipeline is run on 3.9, which could be a painful workflow.  This might not be too much of a big deal in practice but the need to ignore 2 ruff rules due to differences between 3.9/3.10 feels like a bit of a sign.  Thoughts?

Reply:
yeah had thought about that so happy to change https://github.com/lvgig/tubular/pull/326/commits/2843ed4263ed83a650a7e329f132ba45b3bab5b4

Those ruff rules are a bit odd, they still get recommended for 3.9 despite erroring